ui/raidboss/common_replacement.js

```diff
@@ -15,13 +15,27 @@ let commonReplacement = {
     ja: 'の封鎖が解かれた',
     ko: '의 봉쇄가 해제되었습니다',
   },
+  '--adds spawn--': {
+    de: '--adds spawn--',
+    fr: '--apparition d\'adds--',
+    ja: '--adds spawn--',
+    cn: '--adds spawn--',
+    ko: '--쫄 소환--',
+  },
   '--center--': {
     de: '--mitte--',
     fr: '--centre--',
     ja: '--センター--',
     cn: '--中央--',
     ko: '--중앙--',
   },
+  '--clones appear--': {
+    de: '--klone erscheinen--',
+    fr: '--apparition des clones--',
+    ja: '--幻影が現れる--',
+    cn: '--clones appear--',
+    ko: '--clones appear--',
```

Comment:
@MaikoTan, @Jaehyuk-Lee, @Akurosia, @r4ha.
I need your help translating these lines, if there is any need. They were not translated in the other files, sorry to bother you again.

Reply:
`de: '--Klone erscheinen--',`

Reply:
`ja: '--幻影が現れる--',`
or
`ja: '--幻影出現--',`

@r4ha what do you think? In my opinion, I prefer the pure Kanji version.

Reply:
`ko: '--분신 소환--',`

Reply:
> `ja: '--幻影が現れる--',`
> or
> `ja: '--幻影出現--',`
> 
> @r4ha what do you think? In my opinion, I prefer the pure Kanji version.

Yea I think you're right!

Reply:
No one for cn translation? Otherwise I will leave the English words. Thank you all.

Reply:
> No one for cn translation? Otherwise I will leave the English words. Thank you all.

This word was in Varis-ex at the first time, right? The CN server was now remain in 5.15 version because coronavirus, so there is no a common translation for now.
I think you can temporary leave it as '--克隆体出现--' for now.

As the ja translation, '--幻影出現--' was the best, thanks @r4ha !.

Reply:
Thank so much.